databricks/koalas/indexing.py

```diff
@@ -210,7 +216,121 @@ def __getitem__(self, key):
             return self._kdf_or_kser.iloc[key]
 
 
-class _LocIndexerLike(_IndexerLike):
+class _LocIndexerLike(_IndexerLike, metaclass=ABCMeta):
+    def _select_rows(
```

Comment:
@ueshin, is it all moving around the codes? If so looks fine. Can we add some docs for each method while we're here?

Reply:
sure, I'll add them soon. thanks!